homeassistant/components/smarty/sensor.py

```diff
@@ -77,6 +81,21 @@ class SmartySensorDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.TIMESTAMP,
         value_fn=get_filter_days_left,
     ),
+    SmartySensorDescription(
+        key="system_state",
+        translation_key="system_state",
+        value_fn=lambda smarty: smarty.system_state,
+    ),
+    SmartySensorDescription(
+        key="software_version",
+        translation_key="software_version",
+        value_fn=lambda smarty: smarty.get_software_version(),
+    ),
+    SmartySensorDescription(
+        key="configuration_version",
+        translation_key="configuration_version",
+        value_fn=lambda smarty: smarty.get_configuration_version(),
+    ),
```

Comment:
What does configuration version mean?

Reply:
I suppose hardware configuration, Salda is not completely clear about this. Their explanation is "Every unit has a configuration version specific to this unit, which is saved in the production line"